intel_extension_for_transformers/llm/runtime/graph/jblas/jblas/jit_base.hpp

```diff
@@ -227,6 +227,26 @@ class JitAmxbf16 : protected JitAmxtile {
   void cvt_fp32_bf16(const Xbyak::Ymm& _bf16, const Xbyak::Zmm& _fp32) { vcvtneps2bf16(_bf16, _fp32); }
 };
 
-class JitAmxint8 : protected JitAmxtile {};
+class JitAmxint8 : protected JitAmxtile {
```

Comment:
there are 2 copies of jblas, can you remove this one?

Reply:
I think @zhenwei-intel will do the migration for us as part of the work of project directory restructure. I'll rebase the branch then.